src/App.tsx

```diff
@@ -28,34 +28,35 @@ const data = {
   ],
 };
 
+const Home = () => (
+  <div>
+    <a href='http://localhost:3000/module/0/emulator/0'>Visit Emulator</a> <br />
+    <a href='http://localhost:3000/module'>Visit Courses</a>
```

Comment:
My understanding is you can do something like [linking to submodules](https://github.com/Project-Reclass/toynet-react/blob/master/src/ModuleList/Module/SubModule/SubModule.tsx#L11) so that it doesn't break when deployed.

Reply:
Yes, inside the `<Router>` tag, it is already being linked to submodules, the` Home` constants are just placeholders for the splash screen, so that when clicking on the link, it will match the router paths and link to the modules. I talked with Scott after work today to refine the routing!